cmd/gossamer/commands/root.go

```diff
@@ -580,15 +576,15 @@ func execRoot(cmd *cobra.Command) error {
 	}
 
 	// load user keys if specified
-	if err := unlockKeystore(ks.Acco, config.BasePath, config.Account.Unlock, password); err != nil {
+	if err := unlockKeystore(ks.Acco, config.DataDir, config.Account.Unlock, password); err != nil {
```

Comment:
```suggestion
	if err := unlockKeystore(ks.Acco, config.ConfigDir, config.Account.Unlock, password); err != nil {
```

Reply:
I consider keys as data and not config. Could you tell me why you think otherwise?

Reply:
the key difference is what you do with it:
- disk full? wipe data, retain keys & config
- borked database? wipe data, retain keys & config

while your point is reasonable, from a user perspective, there is no reason to have keys in a directory that might be subject to regular purges